src/Bridges/soctopsdbridge.jl

```diff
@@ -11,26 +6,26 @@ Builds a VectorAffineFunction representing the upper (or lower) triangular part
 [ f[2:end] g * I     ]
 """
 function _SOCtoPSDCaff(f::MOI.VectorAffineFunction{T}, g::MOI.ScalarAffineFunction{T}) where T
-    dim = length(f.constant)
+    dim = MOIU.moilength(f)
```

Comment:
Uh, what is `moilength`? `output_length` maybe?

Reply:
I was also looking for a better name while doing this PR :-P
I have created an issue https://github.com/JuliaOpt/MathOptInterface.jl/issues/366